src/components/FLink.vue

```diff
@@ -80,12 +78,12 @@ component.FLink(
     &:hover
       color var(--flink--hover-color)
 
-  &[tabindex]:not([tabindex="-1"]):focus-visible
+  &[tabindex]:not([tabindex='-1']):focus-visible
     outline-offset rem(4)
     outline solid rem(3) currentColor
     transition $outline-transition, color 0.25s
 
-    @media (prefers-reduced-motion)
+    @media (prefers-reduced-motion )
```

Comment:
😬 which formatter did that ?
```suggestion
    @media (prefers-reduced-motion)
```